Source/Scene/Cesium3DTileStyle.js

```diff
@@ -246,13 +246,24 @@ define([
                 return this._show;
             },
             set : function(value) {
+                if (typeof value === 'boolean') {
+                    this._show = new Expression(String(value));
+                } else if (typeof value === 'string') {
+                    this._show = new Expression(value);
+                } else if (defined(value.conditions)) {
+                    this._show = new ConditionsExpression(value);
+                } else if (defined(value.expression) || defined(value.conditionsExpression)) {
+                    this._show = value;
+                } else {
+                    this._show = undefined;
+                    return;
+                }
```

Comment:
Maybe it's not necessary to split up these two last cases. This would also allow someone to define their own evaluator like the example in the documentation.

```javascript
if (typeof value === 'boolean') {
    this._show = new Expression(String(value));
} else if (typeof value === 'string') {
    this._show = new Expression(value);
} else if (defined(value.conditions)) {
    this._show = new ConditionsExpression(value);
} else {
    this._show = value;
}
```